### PR TITLE
Ewl-535 add player infos component

### PR DESCRIPTION
**Pull request by Xb0X**

### CODE REVIEW

File: src/components/PlayerCard/PlayerInfos.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    firstName: PropTypes.string,
+    lastName: PropTypes.string,
+    age: PropTypes.string,
+    height: PropTypes.string,
+    weight: PropTypes.string,
+    country: PropTypes.string,
+    flag: PropTypes.string,
+    ranking: PropTypes.string,
+    organisation: PropTypes.string,
+    picture: PropTypes.string,
```
Comment (davidgarry) [inline src/components/PlayerCard/PlayerInfos.js]: pîctureUrl ?

File: src/components/PlayerCard/PlayerInfos.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  lastName: 'Nadal',
+  age: '32',
+  height: '1.85',
+  weight: '85',
+  country: 'Spain',
+  flag: 'https://i.eurosport.com/_iss_/geo/country/flag/medium/2203.png',
+  ranking: '2',
+  organisation: 'ATP',
+  picture: 'https://i.eurosport.com/_iss_/person/pp_clubteam/large/435121.jpg',
+  background: 'raquette.jpg',
```
Comment (davidgarry) [inline src/components/PlayerCard/PlayerInfos.test.js]: background (backgroundUrl ?) is not defined in props

Comment (Xb0X) [reply src/components/PlayerCard/PlayerInfos.test.js]: Removed, not used anymore. The background will be in the playercard.

File: src/components/PlayerCard/PlayerInfos.js (hunk 1 above)
Comment (Xb0X) [reply src/components/PlayerCard/PlayerInfos.js]: Done

File: src/components/PlayerCard/PlayerInfos.js [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+PlayerInfos.propTypes = {
+  player: PropTypes.shape({
+    firstName: PropTypes.string,
+    lastName: PropTypes.string,
+    age: PropTypes.string,
+    height: PropTypes.string,
+    weight: PropTypes.string,
+    country: PropTypes.string,
+    flag: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (Xb0X) [reply src/components/PlayerCard/PlayerInfos.js]: Done

File: src/components/PlayerCard/PlayerInfos.js [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+    ranking: PropTypes.string,
+    organisation: PropTypes.string,
```
[the remark this answers is not in the document]
Comment (Xb0X) [reply src/components/PlayerCard/PlayerInfos.js]: Done

File: src/index.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ import Adobe from './components/Adobe';
 import DataLayer from './components/DataLayer';
 import Quantcast from './components/Quantcast';
 import GoogleTagManager from './components/GoogleTagManager';
+import PlayerInfos from './components/PlayerCard';
```
[the remark this answers is not in the document]
Comment (Xb0X) [reply src/index.js]: Refactored ;)

File: src/components/PlayerInfos/index.js [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+    firstName: '-',
+    lastName: '-',
+    age: '-',
+    height: '-',
+    weight: '-',
+    country: '-',
+    flagUrl: '-',
+    ranking: '-',
+    competition: '-',
+    pictureUrl: '-',
```
Comment (davidgarry) [inline src/components/PlayerInfos/index.js]: Maybe have a default picture url ?

File: src/components/PlayerInfos/index.js [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+
+PlayerInfos.defaultProps = {
+  player: {
+    firstName: '-',
+    lastName: '-',
+    age: '-',
+    height: '-',
+    weight: '-',
+    country: '-',
+    flagUrl: '-',
```
Comment (davidgarry) [inline src/components/PlayerInfos/index.js]: Maybe have a default flag url ?

**Pull request merged**